float64[4] poly_x
float64[4] poly_y
float64 delta_time